src/go/k8s/main.go

```diff
@@ -84,6 +84,7 @@ func main() {
 	flag.BoolVar(&allowPVCDeletion, "allow-pvc-deletion", false, "Allow the operator to delete PVCs for Pods assigned to failed or missing Nodes (alpha feature)")
 	flag.BoolVar(&redpandav1alpha1.AllowConsoleAnyNamespace, "allow-console-any-ns", false, "Allow to create Console in any namespace. Allowing this copies Redpanda SchemaRegistry TLS Secret to namespace (alpha feature)")
 	flag.StringVar(&restrictToRedpandaVersion, "restrict-redpanda-version", "", "Restrict management of clusters to those with this version")
+	flag.StringVar(&redpandav1alpha1.SuperUsersPrefix, "superusers-prefix", "", "Prefix to add in name of superusers managed by operator")
```

Comment:
since this flag does not change anything for existing users - as we discussed on slack please add this into description here and possibly also add a log line (but if that log line would be logged on every reconciliation loop should probably be a debug?)

Reply:
Updated